gcc/config/riscv/riscv.md

```diff
@@ -169,7 +169,7 @@
 ;; D2S          double to float single
 ;; S2D          float single to double
 
-(define_attr "cnv_mode" "unknown,I2S,I2D,S2I,D2I,D2S,S2D" 
+(define_attr "cnv_mode" "unknown,I2S,I2D,S2I,D2I,D2S,S2D"
```

Comment:
Clean up should split into another patch instead of mix with feature implementation patch.